components/SentenTree/index.js

```diff
@@ -0,0 +1,57 @@
+import VisComponent from '../../VisComponent';
+import { SentenTreeBuilder,
+         SentenTreeVis } from 'sententree/dist/SentenTree';
+
+export default class SentenTree extends VisComponent {
+  static get options () {
+    return [
+      {
+        name: 'data',
+        description: 'The data table.',
+        type: 'table',
+        format: 'objectlist'
+      },
+      {
+        name: 'id',
+        description: 'The field containing the identifier of each row.',
+        type: 'string',
+        domain: {
+          mode: 'field',
+          from: 'data',
+          fieldTypes: ['string', 'integer', 'number']
+        }
+      },
+      {
+        name: 'text',
+        description: 'The field containing the text sample.',
+        domain: {
+          mode: 'field',
+          from: 'data',
+          fieldTypes: ['string']
+        }
+      },
+      {
+        name: 'count',
+        description: 'The field containing the count for each text sample.',
+        type: 'string',
+        domain: {
+          mode: 'field',
+          from: 'data',
+          fieldTypes: ['integer']
+        }
+      }
+    ];
+  }
+
+  constructor (el, {data, graphs = 3}) {
```

Comment:
Good ideas. Check out 3c69e3d and 59bec69. In particular, did I get `type` and `format` right for the `graphs` option?